README.md

```diff
@@ -158,28 +158,30 @@ require'lualine'.get_config()
 ### Starting lualine
 
 ```lua
-require('lualine').setup()
+require'lualine'.setup()
```

Comment:
I think we should use parenthesis on all function calls . function calls without parans may confuse people that are new to lua .

Reply:
That was the case for me too, it's just more natural. I'll update the calls, as suggested.